apps/server/src/shared/repo/user/user.repo.ts

```diff
@@ -163,6 +163,13 @@ export class UserRepo extends BaseRepo<User> {
 		return promise;
 	}
 
+	async deleteUser(userId: EntityId): Promise<number> {
+		const deletedUser: Promise<number> = this._em.nativeDelete(User, {
+			id: userId,
+		});
+		return deletedUser;

```

Comment:
I would suggest using some more meaningful name for this _const_ here like, e.g., `deletedUserCount` or `deletedUserNumber` as it doesn't really contain any deleted user data, just a number of deleted documents.

Reply:
chnaged